app/src/main/java/org/simple/clinic/enterotp/BruteForceOtpEntryProtection.kt

```diff
@@ -18,26 +18,36 @@ class BruteForceOtpEntryProtection @Inject constructor(
     private val config: Observable<BruteForceOtpEntryProtectionConfig>,
     private val otpStatePreference: Preference<BruteForceOtpProtectionState>
 ) {
-
   sealed class ProtectedState {
     data class Allowed(val attemptsMade: Int, val attemptsRemaining: Int) : ProtectedState()
     data class Blocked(val attemptsMade: Int, val blockedTill: Instant) : ProtectedState()
   }
 
-  fun incrementFailedOtpAttempt() {
-    val updatedBruteForceOtpProtectionState = updateFailedOtpAttemptLimitReached(config.limitOfFailedAttempts)
-    otpStatePreference.set(updatedBruteForceOtpProtectionState)
+  fun incrementFailedOtpAttempt(): Completable {
```

Comment:
This can be a synchronous call. Is there a reason we changed it back to return `Completable`? 🤔